### PR TITLE
feat: risk checkbox

### PR DESCRIPTION
**Pull request by defispartan**
When borrowing or withdrawing below a HF of 1.5: 
- Display warning message and checkbox for users to acknowledge risk
- Disable borrow or withdraw button if checkbox is displayed and not accepted

Also reduced the minimum HF on withdraw from 1.07 -> 1.01. The reasoning for this is that if a user has a HF below 1.07 and can't repay from their wallet balance or collateral, then they are effectively locked into their position with the 1.07 limit.

### CODE REVIEW

**Comment by defispartan** [comment]
@iamanastasia

File: src/components/transactions/Borrow/BorrowModalContent.tsx [hunk 1]
```diff
@@ -141,6 +142,7 @@ export const BorrowModalContent = ({
     ),
     currentLiquidationThreshold: user.currentLiquidationThreshold,
   });
+  const displayRiskCheckbox = newHealthFactor.toNumber() < 1.5;
```
Comment (sakulstra) [inline src/components/transactions/Borrow/BorrowModalContent.tsx:145]: Doesn't this need to also check for `!= -1`?

File: cypress/support/steps/main.steps.ts [hunk 2]
```diff
@@ -75,11 +74,13 @@ export const borrow = (
     amount,
     apyType,
     hasApproval = true,
+    isRisk = false,
```
Comment (foodaka) [inline cypress/support/steps/main.steps.ts:77]: what is isRisk?

File: cypress/support/steps/main.steps.ts [hunk 3]
```diff
@@ -115,8 +116,14 @@ export const borrow = (
     it(`Borrow ${amount} amount for ${_shortName}`, () => {
       setAmount({
         amount,
-        hasApproval,
       });
+    });
+    if (isRisk) {
+      it(`Click risk checkbox`, () => {
```
Comment (foodaka) [inline cypress/support/steps/main.steps.ts:122]: a risk checkbox

File: cypress/support/steps/main.steps.ts (hunk 2 above)
Comment (foodaka) [reply cypress/support/steps/main.steps.ts:77]: maybe rename this to something like acceptsLiquidationRisk ?

**Pull request merged**